scipy/signal/_short_time_fft.py

```diff
@@ -420,6 +415,15 @@ class ShortTimeFFT:
     _fac_mag: float | None = None
     _fac_psd: float | None = None
     _lower_border_end: tuple[int, int] | None = None
+    # The following tuples store parameter(s) and return value(s) of methods for caching
+    # (initialized with invalid parameters; should only be accessed by atomic
+    # read/writes to alleviate potential multithreading issues):
+    _cache_post_padding: tuple[int, tuple[int, int]] = -1, (0, 0)
+    _cache_upper_borde_begin: tuple[int, tuple[int, int]] = -1, (0, 0)
```

Comment:
`_cache_upper_borde_begin`

Spellling.

Reply:
Fixed in commit [5f65699](https://github.com/scipy/scipy/pull/23457/commits/5f65699b186faf92691996c444c6349676f34009). Thanks you for the very prompt feedback.